staff/oriol-bover/horroy/scripts/horroy.js

```diff
@@ -0,0 +1,328 @@
+function Horroy() {
+    this.length = arguments.length;
+
```

Comment:
You can control when the user enters an integer, it means you have to create an empty Horroy with that length. 🤓